src/gc/impl/conservative/gc.d

```diff
@@ -122,6 +122,9 @@ __gshared long extendTime;
 __gshared long otherTime;
 __gshared long lockTime;
 
+// thread-local counter
+size_t bytesAllocated;
```

Comment:
I think `ulong` is better here; `size_t` is for sizes of objects in memory, whereas `bytesAllocated` is a tally. It could go over 4 GB on a 32-bit system.